docs-md/app/etl.md

```diff
@@ -0,0 +1,608 @@
+
+We’ve also set up MongoDB as a replica set named `my-replica-set` Debezium requires that MongoDB run in this configuration to pick up changes from its oplog. In this case, we’re just running a single node replica set.
+
+Finally, note that the ksqlDB server image mounts the `confluent-hub-components` directory, too. The jar files that we downloaded need to be on the classpath of ksqlDB when the server starts up.
+
+Bring up the entire stack by running:
+
+```
+docker-compose up
+```
+
+### Create the customers table in Postgres
+
+It's pretty common for companies to keep their customer data in a relational database. Let's model that information in a Postgres table. Start by logging into the container:
+
+```
+docker exec -it postgres /bin/bash
+```
+
+Log into Postgres as the user created by default:
+
+```
+psql -U postgres-user customers
+```
+
+Create a table that represents the customers. For simplicity's sake, we'll just model this with three columns: an id, a name, and the age of the person:
+
+```sql
+CREATE TABLE customers (id TEXT PRIMARY KEY, name TEXT, age INT);
+```
+
+Seed the table with some initial data:
+
+```sql
+INSERT INTO customers (id, name, age) VALUES ('5', 'fred', 34);
+INSERT INTO customers (id, name, age) VALUES ('7', 'sue', 25);
+INSERT INTO customers (id, name, age) VALUES ('2', 'bill', 51);
+```
+
+### Configure MongoDB for Debezium
+
+Now that Postgres is setup, let's configure MongoDB. Start by logging into the container:
+
+```
+docker exec -it mongo /bin/bash
+```
+
+Log into the Mongo console using the username and password specified in the Docker Compose file:
+
+```
+mongo -u mongo-user -p mongo-pw admin
+```
+
+Because MongoDB has been started as a replica set, it needs to be initiated. Run the following command to kick it off:
+
+```javascript
+rs.initiate()
+```
+
+Now that this node has become the primary in the replica set, we need to configure access so that Debezium can replicate changes remotely. Switch into the `config` database:
+
+```
+use config
+```
+
+Create a new role for Debezium. This role will enable the user that we create to access system-level collections, which are normally restricted:
+
+```javascript
+db.createRole({
+    role: "dbz-role",
+    privileges: [
+        {
+            resource: { db: "config", collection: "system.sessions" },
+            actions: [ "find", "update", "insert", "remove" ]
+        }
+    ],
+    roles: [
+       { role: "dbOwner", db: "config" },
+       { role: "dbAdmin", db: "config" },
+       { role: "readWrite", db: "config" }
+    ]
+})
+```
+
+Switch into the `admin` database. We need to create our user here so that it can be authenticated:
+
+```
+use admin
+```
+
+Create the user for Debezium. This user has `root` on the `admin` database, and can also access other databases needed for replication:
```

Comment:
```suggestion
Create the user for Debezium. This user has `root` on the `admin` database, and it can also access other databases needed for replication:
```